resolvers.ts

```diff
@@ -831,7 +834,7 @@ export const resolvers: Resolvers = {
           context,
         });
         return workflowRuns.map(run => ({
-          id: run.id,
+          id: run.id.toString(),
```

Comment:
Does this change need to be made to `workflowRuns` as well? (I guess this would only apply during the period between the Fed deploy and webapp deploy)

Reply:
Or if this is only going to get merged after the staging cut, doesn't matter

Reply:
I think we might not want this change for now because the frontend changes won't go in until after this is deployed.  

Right now, BulkDownloadModal takes what this returns and then sends it to `BulkDownloadModalMutation` -> workflowRunIds ([Int]) , so wouldn't changing this to a string be a problem?